test/support/fixtures/arenas_fixtures.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Sports.ArenasFixtures do
+  @moduledoc """
+  This module defines test helpers for creating
+  entities via the `Sports.Arenas` context.
+  """
+
+  @doc """
+  Generate a arena.
+  """
+  def arena_fixture(attrs \\ %{}) do
```

Comment:
Didn't understand the use of this helper function

Reply:
its for fixtures . we can leave it as it is .. we don't need this instead we are using factories